### PR TITLE
[OSDEV-1383] Approved facility claim contributor notice copy update

### PR DESCRIPTION
**Pull request by FrancescaOSH**
Updated the email text, added links and bolded keywords.

### CODE REVIEW

**Review by roman-stolar — DISMISSED** [review]
LGTM

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-1492](https://opensupplyhub.atlassian.net/browse/OSDEV-1492) - Fixed an issue where invalid manually entered dates were not validated on the UI, resulting in API errors with message “The request query is invalid.” on `Moderation Queue` page. Invalid dates are now trimmed and properly handled.
 
 ### What's new
+* [OSDEV-1383](https://opensupplyhub.atlassian.net/browse/OSDEV-1383) - Edited text of the automated email that notifies a contributor when one of their facilities has been claimed.
```
Comment (vladsha-dev) [inline doc/release/RELEASE-NOTES.md:25]: Hi @FrancescaOSH,
Could you please provide a brief explanation of why you made this update? Specifically, what value do these changes bring?

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: doc/release/RELEASE-NOTES.md (hunk 1 above)
Comment (FrancescaOSH) [reply doc/release/RELEASE-NOTES.md]: Hi @vladsha-dev I have updated the text in the release notes. Let me know if that's enough.

**Review by vladsha-dev — APPROVED** [review]
LGTM

File: doc/release/RELEASE-NOTES.md (hunk 1 above)
Comment (vladsha-dev) [reply doc/release/RELEASE-NOTES.md]: Nice, thank you!

**Pull request merged**